doc/faq.md

```diff
@@ -20,7 +20,9 @@ There are additional requirements to run Selenium based tests:
 1. Download [geckodriver](https://github.com/mozilla/geckodriver/releases)
 1. Unarchive it
 1. Make sure that geckodriver executable in PATH
+1. Create superuser using `python manage.py createsuperuser` with username "walrus" and password "wal"
```

Comment:
First of all properly **rebase** your changes on top of the current master. CI should fail because it automatically executes Selenium test.
Add superuser creation to CI.

Reply:
Done in #98